baselines/teams.md

```diff
@@ -381,19 +423,20 @@ available in GCC or DoD tenants.
 
 ### Implementation
 
-To ensure that teams email integration is disabled:
+All the settings in this section are configured in the **Microsoft Teams admin center**.
```

Comment:
For similar statements made for MS.TEAMS.1 through MS.TEAMS.5, "Microsoft Teams admin center" was a link. Recommend changing this to a link for consistency.

Reply:
Updated to add a link.